scripts/run_with_docker.sh

```diff
@@ -4,7 +4,7 @@ DOCKER_IMAGE_NAME=notifications-template-preview
 source environment.sh
 
 if [[ "${@}" == "web" || "${@}" == "web-local" ]]; then
-  EXPOSED_PORTS="-e PORT=6013 -p 6013:6013"
+  EXPOSED_PORTS="-e PORT=6013 -p 127.0.0.1:6013:6013"
```

Comment:
Why doesn't the value sent to the `-p` flag use the `POST` env var? Maybe it can't but otherwise it would be a bit neater.

Reply:
Looks like we just didn't set `PORT` as an environment variable for this app, we hardcoded it instead. We're quite inconsistent with how we do things, for example for some apps we use an environment variable for the port, others (like here) we don't, and in antivirus this line lives in the Makefile instead of the script to run with Docker.

I didn't attempt to make them all consistent, but I could have a look at doing that.

Reply:
Now updated to set the `PORT` on line 3 so it's easier to see at a glance which port we're using